components/comments/commentList.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import CommentItem from './commentItem';
+
+const CommentList = ({ value }) => {
+    return (
+        <>
+            <div className="row">
+                <div className="col">
+                    <h2>Comments</h2>
+                </div>
+                <div className="col text-right">
+                    <a
+                        href="#"
+                        className="button button-md button-primary button-leaf"
+                        id="addComment"
+                    >
+                        <span className="mdi mdi-plus"></span> Comment
+                    </a>
+                </div>
+            </div>
+            {value.map((comment) => {
+                return <CommentItem comment={comment} key={comment.identifier} />;
+            })}
+        </>
+    );
+};
+
+CommentList.propTypes = {
+    comment: PropTypes.shape({
```

Comment:
Like for example: what a comment properties are?

Reply:
mmm, at this point in this component I don't need anything else than `identifier` attr, on child component `CommentItem` there is the specification of the attributes needed!

![image](https://user-images.githubusercontent.com/37185433/69761408-ed8e9880-112c-11ea-9c0a-106bc82b05b1.png)

so do you think this is still needed at this point?